Core/Core/SwiftGen/Assets.swift

```diff
@@ -111,6 +112,7 @@ public enum CoreAssets {
   public static let noWifiMini = ImageAsset(name: "noWifiMini")
   public static let notAvaliable = ImageAsset(name: "notAvaliable")
   public static let playVideo = ImageAsset(name: "playVideo")
+  public static let settingsIcon = ImageAsset(name: "settingsIcon")
```

Comment:
Since we don't use the word `icon` for icons, how about just renaming it to `settings`?